templates/nginx-ingress.yaml

```diff
@@ -0,0 +1,48 @@
+{{- if .Values.nginxingress.enabled -}}
```

Comment:
It's not clear to me why we need to add (almost) the same file in 2 different repos ([other one](https://github.com/greenpeace/planet4-helm-wordpress/pull/15/files#diff-6e731c85a365c74a329c17d019ee10d243eb3ae65f99ad8ff8fb006a6335f7e4)). But I'm still reading up so could be there's a good reason. In any case it seems better if we can somehow have it in one place.

Reply:
I can probably add a comment here. While it is possible to generalise helm charts and have sub-charts etc. (really generalising and preventing repetition was the whole point of helm) it doesn't make sense for small resources (ie a single manifest) unless there is serious reuse along with other drawbacks below.

These 2 charts serve different purposes (static vs WordPress) and may operate in different ways - all the other resources are declared in their respective repository. Abstracting this resource could have the following impacts:

*  Couple the Wordpress chart and the Static chart too close together (ie. they share further dependencies when we may want to customise one without impacting the other)
* Add confusion to the layout of the charts and impact readability (the savings from not repeating this resource outweigh the confusion generated by abstracting it in a non-standard way - for which we don't have any structures or paradigms in place already)

Reply:
Agree with Gus about the duplication, these 2 charts are for different deployment types.  Also since there is only a few deployments that actually use this chart it is easier to set the TLS requirements in the deployment code.